README.md

```diff
@@ -141,7 +141,7 @@ Add to your `.babelrc`'s plugins array.
 ## Benchmarks
 > Bootstrap: `npm run bootstrap`
 
-> Running the benchmarks: `./scripts/benchmark.js file.js`
+> Running the benchmarks: `./scripts/benchmark.js package` for the main file **or** `./scripts/benchmark.js package path/file.js` for a specific file in the package
```

Comment:
Yes!  I avoided doing it though because they hadn't done it, but will change it.